common/src/main/java/com/wynntils/models/raid/RaidModel.java

```diff
@@ -151,9 +208,151 @@ public void onWorldStateChange(WorldStateEvent event) {
         }
     }
 
+    // Process raid rewards
+    @SubscribeEvent
+    public void onScreenInit(ScreenInitEvent.Pre e) {
+        if (Models.Container.getCurrentContainer() instanceof RaidRewardChestContainer raidRewardChest) {
+            expectedRaidRewardChestId = raidRewardChest.getContainerId();
+            rewardChestIsOpened = true;
+            hasProcessedRewards = false;
+        } else {
+            expectedRaidRewardChestId = -2;
+        }
+    }
+
+    @SubscribeEvent
+    public void onSetContent(ContainerSetContentEvent.Post event) {
+        if (event.getContainerId() != expectedRaidRewardChestId) return;
+
+        if (hasProcessedRewards) {
+            /* Can include logic to handle scrolling through received aspects */
```

Comment:
What does this mean?

Reply:
Right now it only looks through the first 5 aspects that show up when you complete a raid, as they're in the chest as soon as you open it. If by some miracle you get 6 mythic aspects, the 6th can only be identified when the player scrolls to the next page.

Reply:
I wonder if it's possible to use the same detection method as lootruns and check the items that are actually displayed above the reward chest when opening it? I imagine tomes can be done that way but maybe not aspects, would have to look into it